config/variables.js

```diff
@@ -102,7 +102,11 @@ const variables = {
       'hyperlinks': 'primary-blue',
       'visited': 'primary-blue',
       'hover': 'primary-navy',
-      'background-color': 'base-white'
+      'background-color': 'base-white',
+      'border': 'secondary-grey',
```

Comment:
Can we call it "input-border?"

Reply:
Oh wait, if we want all borders within this utility to use this color, leaving it as "border" makes sense.